### PR TITLE
Add policyPath config.

### LINKED ISSUE #12: Read the policy file and import the policies into the database.
I think we should provide a new configuration so that users can import policy data into the database by reading the policy file.

### PR DESCRIPTION
**Pull request by shy1st**
Fix: #12

### CODE REVIEW

**Comment by hsluoyz** [comment]
Is this still in use?

image

**Comment by shy1st** [comment]
@hsluoyz I have deleted it.

File: src/test/resources/model/rbac_policy.csv [hunk 1]
```diff
@@ -1,5 +1,2 @@
-p, alice, data1, read
-p, bob, data2, write
-p, data2_admin, data2, read
-p, data2_admin, data2, write
-g, alice, data2_admin
\ No newline at end of file
+p, alice, /data1, GET
+p, bob, /data2, POST
```
Comment (hsluoyz) [inline src/test/resources/model/rbac_policy.csv:2]: why don't use a classic RBAC policy? https://github.com/jcasbin/jdbc-adapter/blob/master/examples/rbac_policy.csv

read -> GET
write -> POST

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/test/resources/model/rbac_policy.csv (hunk 1 above)
Comment (shy1st) [reply src/test/resources/model/rbac_policy.csv:2]: Because I want to test the currently logged in user have access to the interface.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**